### ISSUE TITLE
In progress: cycling speed and cadence worker

### ISSUE THREAD

**Issue by westenberg** [zewelor/bt-mqtt-gateway#206]
**Cycling speed and cadence (CSC) **
Just to let you know, I'm finishing this worker in the afternoons this week. This can be used to read data from home trainers, desk bikes, bike trackers that correctly implement the bluetooth csc specs: https://www.bluetooth.com/wp-content/uploads/Sitecore-Media-Library/Gatt/Xml/Characteristics/org.bluetooth.characteristic.csc_measurement.xml

I'm using it to get data from this type of desk bike: https://www.deskbike.com

**Functions and features**
It will try to connect based on the set interval. So just start cycling and it will automatically connect and process the data on the background.

Additionally you can pass wheel circumference, calories per hour and it will add meters cycled and calories burned as well.

**Comment by zewelor** [comment]
Nice ! Also this bike look great.

**Comment by jdar** [comment]
Subscribe! This feature looks like exactly what I'm looking for.

**Comment by sirhopcount** [comment]
Any updates on this?

**Comment by lwestenberg** [comment]
No, not planning to develop it anymore. Maybe I will create it for ESPHome <-> Home Assistant since that has excellent Bluetooth support now (for ESP32 devices)
